src/image-resizer/templates/pdb.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.podDisruptionBudget.enabled }}
 ---
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
have you checked the new api spec is compatible in terms of fields? do we need to chance anything?

Reply:
It says an empty `spec.selector` will now select all pods.   But we have a selector.  So I think it is fine.

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125